src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -122,7 +136,7 @@ public function getName()
      * @param OutputInterface $output
      * @param Question        $question
      *
-     * @return bool|mixed|null|string
+     * @return mixed
```

Comment:
same as `ask()`  but as that's a private method i'm not sure we need to completely document it (see [@nicolas-grekas's comment](https://github.com/symfony/symfony/pull/19699#discussion_r75666900)).